orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -386,6 +412,54 @@ at::Tensor& zero_(at::Tensor& self){
   return self;
 }
 
+// aten::add_.Tensor(Tensor(a!) self, Tensor other, *, Scalar alpha=1) -> Tensor(a!)
```

Comment:
could you add a comment here like "TODO: enhance the genop script to support inplace binary operations."

Reply:
Added